engine/src/flutter/shell/platform/android/flutter_main.cc

```diff
@@ -85,6 +81,10 @@ const flutter::Settings& FlutterMain::GetSettings() const {
   return settings_;
 }
 
+flutter::AndroidRenderingAPI FlutterMain::GetAndroidRenderingAPI() {
+  return android_rendering_api_;
```

Comment:
Still not written anywhere?

Reply:
I have not mastered the arts of git push apparently